### DISCUSSION TITLE
Nova show soft deleted relationship as default

### DISCUSSION THREAD

**Discussion by meyer59** [laravel/nova-issues#4315] [Q&A] [answered]
- Laravel Version: 9.14
- Nova Version: 4.6.1
- PHP Version: 8.1
- Database Driver & Version: Mysql 8
- Operating System and Version: Win10
- Browser type and version: Chrome 101 

### Description:
Hello
Every model in our Laravel use Soft delete. 
We have a resource with HasOne relationship. Since Nova 4, when we delete the HasOne we cannot create another one. In short, nova force us to alter/restore the old HasOne instead of creating a new one (picture below). We would like to consider the old one as **deleted** and create a new one. 
Is there anything to change to have the old behavior ? 
![image](https://user-images.githubusercontent.com/9505463/170291301-6f75a126-36b1-4135-b5f3-12d974b5f0a3.png)

### Detailed steps to reproduce the issue on a fresh Nova installation:
Delete an hasOne relationship

**Comment by crynobone** [comment, marked as answer]
It seems you should be using `HasMany` and a combination of `HasOne::ofMany` field instead.

**Comment by meyer59** [comment]
I use the `->hasOne()->latestOfMany()` relationship in my model. I dont want to use the `HasMany ` since the latest model is relevant here, the other is only for record purposes.
If i use the` HasOne::ofmany` nova fiels i cannot create a new model. 
This is not solving the issue here
